### PR TITLE
Update URLs and fix some UI

### PR DESCRIPTION
**Pull request by ischaojie**
1. Fix cloud-sync url
2. Fix some UI with `overflow-scroll` -> `scroll-y-auto`
3. Change close button

Before:

image

After:

image

### CODE REVIEW

**Comment by GeoffreyChen777** [comment]
May I ask which OS are you using? It seems that you are using macOS according to your libfolder path. But the overflow on my own Mac looks ok, bug obviously not in yours. I want to know why.

File: app/renderer/ui/preference-view/preference-view.vue
Comment (GeoffreyChen777) [inline app/renderer/ui/preference-view/preference-view.vue:1]: About the close btn, I did try to put it at the top-right corner before. But I feel that it looks weird and the style of a circle button is inconsistent with the style of other UI components as others are almost rectangles with rounded corners.

I think currently we just keep the previous design would be better.

**Comment by ischaojie** [comment]
🤔 my device is a MacBook Pro M1, Sonoma 14.2.1.

File: app/renderer/ui/preference-view/preference-view.vue
Comment (ischaojie) [reply app/renderer/ui/preference-view/preference-view.vue:1]: ok

**Comment by GeoffreyChen777** [comment]
emmm. I have a M1 mini with Sonoma and a Ventura MBP. Both look good without any issue. I need to investigate more about this…

Anyway, appreciate your fixing PR.

**Pull request merged**